text/0000-else-match.md

```diff
@@ -0,0 +1,245 @@
+- Feature Name: Else Match
+- Start Date: 2016-07-26
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Extend the `if` expression to accept an `else match` block.
+
+# Motivation
+[motivation]: #motivation
+
+This proposal is meant to reduce the verbosity of writing `if ... else { match ... } ` style
+statements.
+
+It also makes code similar to:
+
+```rust
+let flag = false;
+
+if foo() {
+    do_this();
+} else {
+    match bar() {
+        baz() => do_that(),
+        _ => flag = true
+    }
+}
+
+if flag {
+    do_something();
+}
+```
+
+simpler and more concise:
+
+```rust
+if foo() {
+    do_this();
+} else match bar() {
+    baz() => do_that()
+} else {
+    do_something();
+}
+```
+
+## Use-cases
```

Comment:
To be honest, in no of these cases I see a clear improvement. In fact, the opposite is true. I find it "uglier".